### PR TITLE
extend parser of ceph_df for ceph nautilus 14.x

### PR DESCRIPTION
**Pull request by pescobar**
the `ceph_df` check was not working because the latest ceph (release name nautilus version number 14.x) changed the output of `ceph df detail`

this fix is working for my ceph cluster running ceph-14.2.4

### CODE REVIEW

File: checks/ceph_df [hunk 1]
```diff
@@ -48,6 +48,10 @@ def parse_ceph_df(info):
             section = "global"
             continue
 
+        elif line[0] == "RAW":
```
Comment (Titan-C) [inline checks/ceph_df]: Include in the previous clause. `line[0] in ['GLOBAL', 'RAW']`

**Comment by pescobar** [comment]
I don't know how to generate the input for the unit tests.

Would it help if I provide an example of the new output of `ceph df detail` in the latest ceph version?

**Comment by Titan-C** [comment]
To generate the input you can do it quite adhoc. Just write in the parse function `print(info)` or write to a file.
The rest of the instructions how to generate this test file are here
https://github.com/tribe29/checkmk/blob/master/tests/unit/checks/generictests/regression.py

**Comment by pescobar** [comment]
I tried to follow the docs but I am stuck in this error when executing ` py.test -T unit tests/unit/checks/generictests/regression.py --datasetfile ~/tmp/example.py`

```
ImportError while loading conftest '/home/pablo/github/checkmk/tests/unit/checks/generictests/conftest.py'.
tests/unit/checks/generictests/__init__.py:61: in <module>
    from generictests.run import run
tests/unit/checks/generictests/run.py:13: in <module>
    from cmk.gui.watolib.rulespecs import rulespec_registry
cmk/gui/watolib/__init__.py:73: in <module>
    import cmk.gui.sites
cmk/gui/sites.py:32: in <module>
    import cmk.gui.config as config
cmk/gui/config.py:55: in <module>
    from cmk.gui.cee.plugins.config.cee import *  # pylint: disable=wildcard-import,unused-wildcard-import
E   ImportError: No module named cee.plugins.config.cee
```

trying `ls -l cmk_base/cee` in my repo clone I can only find a broken symlink to `../enterprise/cmk_base/cee` . It seems that those libraries are only available in the cmk enterprise edition?

I post here the `example.py` file I generated doing a `print info` at the end of the function `parse_ceph_df()` in case you can create the unit test

```
checkname = 'ceph_df'

info = [[u'RAW', u'STORAGE:'], [u'CLASS', u'SIZE', u'AVAIL', u'USED', u'RAW', u'USED', u'%RAW', u'USED'], [u'ssd', u'84', u'TiB', u'81', u'TiB', u'2.9', u'TiB', u'3.0', u'TiB', u'3.52'], [u'TOTAL', u'84', u'TiB', u'81', u'TiB', u'2.9', u'TiB', u'3.0', u'TiB', u'3.52'], [u'POOLS:'], [u'POOL', u'ID', u'STORED', u'OBJECTS', u'USED', u'%USED', u'MAX', u'AVAIL', u'QUOTA', u'OBJECTS', u'QUOTA', u'BYTES', u'DIRTY', u'USED', u'COMPR', u'UNDER', u'COMPR'], [u'glance-images', u'1', u'25', u'GiB', u'5.88k', u'75', u'GiB', u'0.10', u'25', u'TiB', u'N/A', u'N/A', u'5.88k', u'0', u'B', u'0', u'B'], [u'cinder-volumes', u'2', u'616', u'GiB', u'158.31k', u'1.8', u'TiB', u'2.32', u'25', u'TiB', u'N/A', u'N/A', u'158.31k', u'0', u'B', u'0', u'B'], [u'nova-vms', u'3', u'349', u'GiB', u'91.08k', u'1.0', u'TiB', u'1.32', u'25', u'TiB', u'N/A', u'N/A', u'91.08k', u'0', u'B', u'0', u'B'], [u'cephfs_data', u'4', u'0', u'B', u'0', u'0', u'B', u'0', u'25', u'TiB', u'N/A', u'N/A', u'0', u'0', u'B', u'0', u'B'], [u'cephfs_metadata', u'5', u'15', u'KiB', u'60', u'969', u'KiB', u'0', u'25', u'TiB', u'N/A', u'N/A', u'60', u'0', u'B', u'0', u'B'], [u'.rgw.root', u'6', u'2.6', u'KiB', u'6', u'288', u'KiB', u'0', u'25', u'TiB', u'N/A', u'N/A', u'6', u'0', u'B', u'0', u'B'], [u'default.rgw.control', u'7', u'0', u'B', u'8', u'0', u'B', u'0', u'25', u'TiB', u'N/A', u'N/A', u'8', u'0', u'B', u'0', u'B'], [u'default.rgw.meta', u'8', u'0', u'B', u'0', u'0', u'B', u'0', u'25', u'TiB', u'N/A', u'N/A', u'0', u'0', u'B', u'0', u'B'], [u'default.rgw.log', u'9', u'0', u'B', u'207', u'0', u'B', u'0', u'25', u'TiB', u'N/A', u'N/A', u'207', u'0', u'B', u'0', u'B']]
```

**Comment by Titan-C** [comment]
Yes, you stumbled on quite an interesting bug regarding imports of the enterprise edition.
I'll take over this PR, thank you.

**Pull request closed without merging**